### PR TITLE
Fix mip rna link

### PR DESCRIPTION
**Pull request by henrikstranneheim**
This PR fixes:

- Broken mip-rna  link command

### How to prepare for test
- [x] ssh to hasta (depending on type of change)
- [x] install on stage:
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh fix_mip_rna_link`

### How to test
- [x] login to hasta
- [x] do `cg workflow mip-rna link -c finequagga`

### Expected test outcome
- [x] check that the command queries HK, but does not find any files
- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [x] code approved by MR
- [x] tests executed by HS
- [x] "Merge and deploy" approved by HS, MR
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
### How to test
- [ ] login to hasta
- [ ] do `cg workflow mip-rna link -c finequagga`

Before:
Screenshot 2021-01-13 at 11 16 46

After:
Screenshot 2021-01-13 at 11 20 36

### Expected test outcome
- [ ] check that the command queries HK, but does not find any files because the fastq files are compressed

File: cg/cli/workflow/mip_rna/base.py [hunk 1]
```diff
@@ -55,11 +55,13 @@ def link(context: click.Context, case_id: str, sample_id: str):
...
-            rna_api.link_sample(
-                sample=link_obj.sample,
-                case_id=case_id,
+        if not link_obj.family.data_analysis:
+            LOG.warning(
+                f"No analysis set for {link_obj.sample.internal_id}, file will still be linked"
             )
+            rna_api.link_sample(sample=link_obj.sample, case_id=link_obj.family.internal_id)
+        if link_obj.family.data_analysis == str(Pipeline.MIP_RNA):
+            rna_api.link_sample(sample=link_obj.sample, case_id=link_obj.family.internal_id)
```
Comment (Mropat) [inline cg/cli/workflow/mip_rna/base.py:64]: Thanks for fixing that! I suppose it broke when we have set strict Pipeline definitions, and was not discovered until much later!

**Review by Mropat — APPROVED** [review]
:sunglasses: :+1:

**Comment by henrikstranneheim** [comment]
Deployed on hasta:
Screenshot 2021-01-14 at 16 04 39

**Pull request merged**